README.md

```diff
@@ -1,37 +1,216 @@
+
+Archives submissions and their results & files into the archive.
+
+```bash
+docker run --name archiver cccs/assemblyline-core python -m assemblyline_core.archiver.run_archiver
+```
+
+### Dispatcher
 
 Route the files in the system while a submission is tacking place. Make sure all files during a submission are completed by all required services.
 
-##### Expiry
+```bash
+docker run --name dispatcher cccs/assemblyline-core python -m assemblyline_core.dispatching
+```
+
+### Expiry
 
 Delete submissions and their results when their TTL expires.
 
-##### Ingester
+```bash
+docker run --name expiry cccs/assemblyline-core python -m assemblyline_core.expiry.run_expiry
+```
+
+### Ingester
 
 Move ingested files from the priority queues to the processing queues.
 
-##### Metrics
+```bash
+docker run --name ingester cccs/assemblyline-core python -m assemblyline_core.ingester
+```
+
+### Metrics
 
 Generates metrics of the different components in the system.
 
-##### Scaler
+#### Heartbeat Manager
+
+```bash
+docker run --name heartbeat cccs/assemblyline-core python -m assemblyline_core.metrics.run_heartbeat_manager
+```
+
+#### Metrics Aggregator
+
+```bash
+docker run --name metrics cccs/assemblyline-core python -m assemblyline_core.metrics.run_metrics_aggregator
+```
+
+#### Statistics Aggregator
+
+```bash
+docker run --name statistics cccs/assemblyline-core python -m assemblyline_core.metrics.run_statistics_aggregator
+```
+
+### Scaler
 
 Spin up and down services in the system depending on the load.
 
-##### Updater
+```bash
+docker run --name scaler cccs/assemblyline-core python -m assemblyline_core.scaler.run_scaler
+```
+
+### Updater
 
 Make sure the different services get their latest update files.
 
-##### Workflow
+```bash
+docker run --name updater cccs/assemblyline-core python -m assemblyline_core.updater.run_updater
+```
+
+### Workflow
 
 Run the different workflows in the system and apply their labels, priority and status.
+
+```bash
+docker run --name workflow cccs/assemblyline-core python -m assemblyline_core.workflow.run_workflow
+```
+
+## Documentation
+
+For more information about these Assemblyline components, follow this [overview](https://cybercentrecanada.github.io/assemblyline4_docs/overview/architecture/) of the system's architecture.
+
+---
+
+# Assemblyline 4 - Core
+
+Ce dépôt fournit des services de base pour Assemblyline 4.
+
+## Variantes d'images et balises
+
+| **Type d'étiquette** | **Description**                                                                                                                    |  **Exemple d'étiquette**   |
+| :------------------: | :--------------------------------------------------------------------------------------------------------------------------------- | :------------------------: |
+|       dernière       | La version la plus récente (peut être instable).                                                                                   |          `latest`          |
```

Comment:
Should be either just in french or english not both. I feel like it should be only in french in this case so it would be "type de production" et "séries".